cmd/root.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/oncall-scheduler/cmd/schedule"
+	"github.com/spf13/cobra"
+)
+
+const (
+	name        = "oncall-scheduler"
+	description = "Tool to schedule oncall shifts"
+)
+
+type Config struct {
+	Logger micrologger.Logger
+	Stderr io.Writer
+	Stdout io.Writer
+
+	BinaryName string
+	GitCommit  string
+	Source     string
+}
+
+func New(config Config) (*cobra.Command, error) {
+	if config.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
+	}
+	if config.Stderr == nil {
+		config.Stderr = os.Stderr
+	}
+	if config.Stdout == nil {
+		config.Stdout = os.Stdout
+	}
+
+	if config.GitCommit == "" {
```

Comment:
~~We should may make use of the `project` package we started to introduce with VOO. See also what Theo announced: https://gigantic.slack.com/archives/C04TGHDEF/p1560515977009100.~~

---

Hohoho I saw you are a good boy and added the `project`. Nice! Then we do not need to configure these things but only use the package interface to get information where they are required.